app/registries/foreman/settings/auth.rb

```diff
@@ -32,11 +32,6 @@
       description: N_('Only known Smart Proxies may access features that use Smart Proxy authentication'),
       default: true,
       full_name: N_('Restrict registered smart proxies'))
-    setting('require_ssl_smart_proxies',
-      type: :boolean,
-      description: N_('Client SSL certificates are used to identify Smart Proxies (:require_ssl should also be enabled)'),
-      default: true,
-      full_name: N_('Require SSL for smart proxies'))
```

Comment:
@ezr-ondrej do we still need a migration to drop this setting from the DB?

Reply:
@ekohl it will stay in DB unless you do, but it's harmless and we can come up with a clean-up rake task to remove all obsolete settings (because we have list of settings valid for current install now).

Opinion comming: it's probably safer to have the migration, but I'd rather not have it, because I hate those (migration for removing one date from DB is IMHO wrong :))

Reply:
I like safe. While it feels like a lot of code just to clean it up, it's what we've always done. So to limit the scope of this PR, I'd go with the safe route.